Cloud Application Manager/Core Concepts/providers.md

```diff
@@ -1,29 +1,54 @@
 "contentIsHTML": false
 }}}
 
+### Providers <!-- omit in toc -->
+
+**In this article:**
+
+* [Overview](#overview)
+* [Audience](#audience)
+* [Prerequisites](#prerequisites)
+* [Providers](#providers)
+* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+
+### Overview
+
+Providers is part of the basics of Cloud Application Manager. This article is meant to assist Cloud Application Manager customers who are starting using the plataform.
+
+### Audience
+
+All Cloud Application Manager customers.
+
+### Prerequisites
+
+An active Cloud Application Manager account.
+
 ### Providers
 
 A provider is a public or private cloud account you register in Cloud Application Manager. Cloud Application Manager interfaces with the provider account’s API to provision and orchestrate deployments automatically. Before you can deploy workloads from Cloud Application Manager, you need to register a provider first. Cloud Application Manager integrates with many popular clouds. Here’s a full list of clouds Cloud Application Manager supports and their availability by Edition:
 
-| Clouds | Enterprise <br> (Saas) | Enterprise <br> (Appliance) |
-|-----|:-----:|:-----:|
-| [CenturyLink Cloud](../Deploying Anywhere/using-centurylink-cloud.md) | ✓ | ✓ |
-| [CenturyLink DCC](../Deploying Anywhere/using-dcc.md) | ✓ | ✓ |
-| [Amazon Web Services](../Deploying Anywhere/using-your-aws-account.md) | ✓ | ✓ |
-| [Google Cloud](../Deploying Anywhere/using-google-cloud.md) | ✓ | ✓ |
-| [Microsoft Azure](../Deploying Anywhere/using-azure.md) | ✓ | ✓ |
-| [VMware vCenter](../Deploying Anywhere/using-the-vmware-vcenter-private-datacenter.md) | ✓ | ✓ |
-| [vCloud Air and vCloud Director](../Deploying Anywhere/orchestrating-vcloud-air-vcloud-director-deployments.md) | ✓ | ✓ |
-| [OpenStack Cloud](../Deploying Anywhere/using-openstack-cloud.md) | ✓ | ✓ |
-| [CloudStack](../Deploying Anywhere/using-cloudstack.md) | ✓ | ✓ |
-| [Rackspace Cloud](../Deploying Anywhere/using-rackspace-cloud.md) | ✓ | ✓ |
-| [SoftLayer](../Deploying Anywhere/using-softlayer.md) | ✓ | ✓ |
-
+| Clouds                                                                                                                | Enterprise <br> (Saas) | Enterprise <br> (Appliance) |
+| --------------------------------------------------------------------------------------------------------------------- | :--------------------: | :-------------------------: |
+| [CenturyLink Cloud](../Deploying%20Anywhere/using-centurylink-cloud.md)                                               | ✓                      | ✓                           |
+| [CenturyLink DCC](../Deploying%20Anywhere/using-dcc.md)                                                               | ✓                      | ✓                           |
+| [CenturyLink Private Cloud on VMware Cloud Foundation](../../centurylink-private-cloud-on-vmware-cloud-foundation.md) | ✓                      |                             |
```

Comment:
Any reason why you removed the tick in Appliance?

Reply:
When I added that service I checked in my local (applicance version) of CAM and couldn't find the provider.